.vscode/settings.json

```diff
@@ -27,4 +27,7 @@
         "tobytes",
         "viam"
     ],
+    "isort.args": [
+        "-l 140 --profile black"
```

Comment:
surprised at some of the changes black is ok with, is it the line length change?

Reply:
Line length and ordering (alphabetical, constants vs everything else)